MarvelApp/MarvelApp/Network/BaseRequest.swift

```diff
@@ -0,0 +1,22 @@
+protocol BaseRequest {
```

Comment:
Here we can create a `method` property that can have all necessary HTTP methods for this request (e.g: `GET, PUT, POST` etc)